Fix falcon castle access to doors and teleport

rigis1 opened the pull request:
# Description
Map issues will be resolved in 
- [x] #759
## Fix
- access to quest doors in falcon castle
- access to teleport in falcon castle
- door to ghoul quest

## Add
- timer and random reward to chests
- reward from chests in falcon castle

## Fixes

#737

## Type of change

  - [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

  - [x] My code follows the style guidelines of this project
  - [x] I have performed a self-review of my own code
  - [x] I checked the PR checks reports
  - [x] My changes generate no new warnings

Sargitarium commented:
Só uma pergunta... com essa configuração, terá que alterar alguma coisa no ITEMS.XML ou já ocorre automaticamente????

MajestyOTBR commented:
PR closed due to inactivity. If you want, you can reopen to finish.

rigis1 committed:
day 1 fix

fixed falcon castle access to doors and teleport
released aid 3255, 57602, 57604, 57699
added timer to chests

rigis1 committed:
day 2 fix

released aid (57601, 57602, 57603, 64007)
add random reward to chests
add chests with random reward

rigis1 committed:
day 2.1 fix

patch due map changes

rigis1 committed:
day 2.2 fix

update to script

rigis1 committed:
day 3 fix

update to scripts
fix husky killcounter
